releases/8.0/ja.php

```diff
@@ -0,0 +1,508 @@
+        <div class="php8-compare__label">PHP 7</div>
+        <div class="php8-code phpcode">
+            <?php highlight_php_trimmed(
+                'strlen([]); // Warning: strlen() expects parameter 1 to be string, array given
+
+array_chunk([], -1); // Warning: array_chunk(): Size parameter expected to be greater than 0'
+            );?>
+        </div>
+      </div>
+      <div class="php8-compare__arrow"></div>
+      <div class="php8-compare__block example-contents">
+        <div class="php8-compare__label php8-compare__label_new">PHP 8</div>
+        <div class="php8-code phpcode">
+            <?php highlight_php_trimmed(
+                'strlen([]); // TypeError: strlen(): Argument #1 ($str) must be of type string, array given
+
+array_chunk([], -1); // ValueError: array_chunk(): Argument #2 ($length) must be greater than 0'
+            );?>
+        </div>
+      </div>
+    </div>
+    <div class="php8-compare__content">
+      <p>ほとんどの内部関数は、引数の検証に失敗すると Error 例外をスローするようになりました。</p>
+    </div>
+  </div>
+</section>
+
+<section class="php8-section php8-section_light">
+  <h2 class="php8-h2">JIT (ジャストインタイム) コンパイル</h2>
+  <p>
+    PHP 8 は JITコンパイル のエンジンをふたつ搭載しています。
+    トレーシングJITは、もっとも有望なふたつの人工的なベンチマークで、
+    約3倍のパフォーマンスを示しました。
+    また、長期間動いている特定のあるアプリケーションでは、1.5-2倍のパフォーマンス向上が見られました。
+    典型的なアプリケーションのパフォーマンスは、PHP 7.4 と同等でした。
+  </p>
+  <h3 class="php8-h3">
+    PHP 8 のパフォーマンスに対するJITの貢献
+    <!--
+    Relative JIT contribution to PHP 8 performance
+    -->
+  </h3>
+  <p>
+    <img src="/images/php8/scheme.svg" width="900" alt="Just-In-Time compilation">
+  </p>
+
+  <div class="php8-columns">
+    <div class="php8-column">
+      <h2 class="php8-h2 php8-h2_margin-top">型システムとエラーハンドリングの改善</h2>
+      <ul>
+        <li>
+          算術/ビット演算子のより厳密な型チェック
+          <a href="https://wiki.php.net/rfc/arithmetic_operator_type_checks">RFC</a>
+        </li>
+        <li>
+          トレイトの抽象メソッドの検証 <a href="https://wiki.php.net/rfc/abstract_trait_method_validation">RFC</a>
+        </li>
+        <li>
+          マジックメソッドのシグナチャ <a href="https://wiki.php.net/rfc/magic-methods-signature">RFC</a>
+        </li>
+        <li>
+          エンジンの警告を整理 <a href="https://wiki.php.net/rfc/engine_warnings">RFC</a>
+        </li>
+        <li>
+          互換性のないメソッドシグナチャは fatal error に。
+          <a href="https://wiki.php.net/rfc/lsp_errors">RFC</a>
+        </li>
+        <li>
+          @ 演算子は、致命的なエラーを隠さなくなりました。
+        </li>
+        <li>
+          private メソッドの継承時のシグナチャチェック <a href="https://wiki.php.net/rfc/inheritance_private_methods">RFC</a>
+        </li>
+        <li>
+          Mixed 型のサポート <a href="https://wiki.php.net/rfc/mixed_type_v2">RFC</a>
+        </li>
+        <li>
+          戻り値で static 型をサポート <a href="https://wiki.php.net/rfc/static_return_type">RFC</a>
+        </li>
+        <li>
+          内部関数に型アノテーション
+          <a href="https://externals.io/message/106522">Email thread</a>
```

Comment:
```suggestion
          <a href="https://externals.io/message/106522">メールスレッド</a>
```
Or something similar